app/controllers/user_stories_controller.rb

```diff
@@ -115,7 +118,7 @@ def user_story_must_exist
   end
 
   def user_story_params
-    params[:user_story].permit(:definition, :story_points, :stakeholder, :cannot_be_estimated, :description, { acceptance_criteria_attributes: [:id, :detail, :_destroy]}, {tasks_attributes: [:id, :definition, :description, :_destroy]})
+    params[:user_story].permit(:lock_version, :definition, :story_points, :stakeholder, :cannot_be_estimated, :description, { acceptance_criteria_attributes: [:id, :detail, :_destroy]}, {tasks_attributes: [:id, :definition, :description, :_destroy]})
```

Comment:
Line is too long. [250/80]<br>Redundant curly braces around a hash parameter.<br>Space inside } missing.<br>Space inside { missing.